legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/helperFunctions/helperFunctions.pure

```diff
@@ -116,6 +116,7 @@ function <<access.private>> meta::alloy::service::execution::splitWithEmptyValue
 
    $s->meta::alloy::service::execution::replaceWithEmptyValue($delimForEmptyCsvField)
         ->meta::alloy::service::execution::splitCsvRow(',')
+        ->map(csv_val| if($csv_val->contains(',') && $csv_val->startsWith('"') && $csv_val->endsWith('"'), | $csv_val->substring(1, $csv_val->length()-1) ,| $csv_val))
```

Comment:
can we add a test for the csv to sql statements to ensure the expected values are carried through?

Reply:
> also confused why we didnt have to update tests from this PR #1988

The test added in PR https://github.com/finos/legend-engine/pull/1988 has a leading white space. We don't remove white space.

Reply:
yup just saw that, lets add a test where we ensure this. 
what about `testRelationalServiceWithCommmaInCSV()` in https://github.com/finos/legend-engine/pull/1988/
would expect that to start failing or change?